common/script/content/index.js

```diff
@@ -1251,7 +1251,7 @@ api.quests = {
       str: 1.5
     },
     drop: {
-      items: [
+      itemsForOwner: [
```

Comment:
The idea was to split the awards of a quest into two different arrays, items for all participants and items for just the quest owner. It's not a rename so much as moving some of the rewards into a new array titled itemsForOwner. 

That being said, it probably will be a breaking change in terms of correctly displaying the quest object on the mobile app. The only other way I can think of distinguishing these is to update the item model and add another field to it that says who it applies to. So something like item.awardsOnlyQuestOwner. The reason I didn't do this in the first place was because it seemed to be less efficient as then getting the list of rewards for allParticipants and for just the quest owner would both require a mapping function on the full list to turn it into the two arrays.

If this would be a breaking change, it's probably worth sacrificing the efficiency in favor of not breaking the apps. Should I update this to the item property model?